package.json

```
@@ -13,57 +13,51 @@
   "author": "Moroine <moroine.bentefrit@beta.gouv.fr>",
   "license": "MIT",
   "engines": {
-    "node": ">=20",
-    "npm": ">=10.0.0",
+    "node": ">=24",
+    "npm": ">=11.0.0",
     "yarn": "please-use-npm"
   },
   "scripts": {
-    "lint": "eslint --ignore-path .gitignore --cache --ext .js,.ts,.cjs .",
+    "lint": "eslint --cache .",
     "lint:fix": "npm run lint -- --fix",
     "prettier:fix": "prettier --write -u .",
     "prettier:check": "prettier --check -u .",
     "release": "semantic-release",
-    "prepare": "husky install",
     "test": "vitest",
     "test:ci": "npm run -- test --run",
     "build": "tsup-node --env.NODE_ENV production",
     "typecheck": "tsc --noEmit"
   },
   "peerDependencies": {
-    "bson": " ^5.0.0 || ^6.0.0",
-    "zod": "^3.22.4"
-  },
-  "dependencies": {
-    "json-schema": "^0.4.0",
-    "zod-to-json-schema": "^3.21.4"
+    "bson": "^6.0.0",
+    "zod": "^3.25.64"
```

```suggestion
    "zod": "^4.0.13"
```
https://github.com/colinhacks/zod/releases/tag/v4.0.13